Makefile

```diff
@@ -40,7 +40,7 @@ migration:
 	go build -o bin/machine-api-migration cmd/machine-api-migration/main.go
 
 unit:
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path --bin-dir $(PROJECT_DIR)/bin --index https://raw.githubusercontent.com/openshift/api/master/envtest-releases.yaml)" ./hack/test.sh "./pkg/... ./assets/..." 5m
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path --bin-dir $(PROJECT_DIR)/bin --index https://raw.githubusercontent.com/openshift/api/master/envtest-releases.yaml)" ./hack/test.sh "./pkg/controllers/machinesetsync/..." 5m
```

Comment:
Don't think this was meant to be committed?

Reply:
Thanks for catching. Fixed.

Reply:
Now it's looking at manifests gen and not assets? What are we expecting this to be?

Reply:
Yes this was pointing to `assets/` which was the old name for the `manifests-gen`, when I refactored I forgot to repoint this, hence I fixed it now.